packages/apollo-client/src/__tests__/local-state/general.ts

```diff
@@ -34,6 +35,38 @@ describe('General functionality', () => {
     });
   });
 
+  // TODO The functionality tested here should be removed (along with the test)
+  // once apollo-link-state is fully deprecated.
+  it('should strip @client fields only if no client resolvers specified', async () => {
```

Comment:
I think this should be:

```suggestion
  it('should strip @client fields only if client resolvers specified', async () => {
```